packages/styling/src/ParticipantView/ParticipantView-layout.scss

```diff
@@ -6,6 +6,7 @@ $scope-name: 'str-video__participant-view';
   width: 100%;
   max-width: 1280px;
   max-height: 100%;
+  min-height: 0;
```

Comment:
Is this a change for a specific layout?

Reply:
This is reaction to the component overflow in messenger clone.

Reply:
I have removed the rule in CallParticipantsView as you requested :)